test/unit/gateways/authorize_net_cim_test.rb

```diff
@@ -342,6 +344,20 @@ def test_should_get_customer_profile_request
     assert_instance_of Response, response
     assert_success response
     assert_equal @customer_profile_id, response.authorization
+
+    @gateway.expects(:ssl_post).returns(successful_get_customer_profile_response)
+
+    assert response = @gateway.get_customer_profile(:email => @email)
+    assert_instance_of Response, response
+    assert_success response
+    assert_equal @email, response.params['email']
+
+    @gateway.expects(:ssl_post).returns(successful_get_customer_profile_response)
```

Comment:
I'd prefer to guard these with `.when()` calls, rather than just adding new `.returns()` calls at specific junctions in tests, but that's a personal preference.